playbook/roles/default_page/files/defaultpage-ingress.yaml

```diff
@@ -1,17 +1,21 @@
 apiVersion: networking.k8s.io/v1
 kind: Ingress
 metadata:
-  name: page-ingress
+  name: defaultpage-ingress
+  namespace: default
   annotations:
-    nginx.ingress.kubernetes.io/rewrite-target: /$1 
+    kubernetes.io/ingress.class: nginx
+    nginx.ingress.kubernetes.io/rewrite-target: /$2
```

Comment:
Could be helptul if you can add a comment such as
"rewring urls such as XXX into YYY""